app/posts/govuk-eleventy-plugin.md

```diff
@@ -0,0 +1,53 @@
+
+With this foundation in place, [the design history](https://bat-design-history.netlify.app/) grew to over 500 posts written by more than 20 authors. In 2020 we made this tool available as [a template repository on GitHub](https://github.com/x-govuk/govuk-design-history) so that other teams could quickly begin writing their own design histories. The tool has since been adopted by other teams, both inside the DfE and across government.
+
+However, as the number of sites using this template has grown, the harder it has become to share improvements. Abstracting the common parts into an npm module makes it easier for us to do this.
+
+## A plugin for Eleventy
+
+The [GOV.UK Eleventy Plugin](https://x-govuk.github.io/govuk-eleventy-plugin/) makes it easy to start writing documentation rather than spend time building a website for it.
+
+It does this by providing [a set of layouts](https://x-govuk.github.io/govuk-eleventy-plugin/layouts/) for different content types and uses [typography classes from the GOV.UK Design System](https://design-system.service.gov.uk/styles/typography/) to ensure documents look familiar to users of other GOV.UK services. (This is achieved via [a plugin for markdown-it](https://github.com/x-govuk/markdown-it-govuk), which can be used on other projects if desired).
+
+Markdown written using [an extended syntax](https://x-govuk.github.io/govuk-eleventy-plugin/markdown-advanced/) is also supported, meaning authors can easily include tables, abbreviations, footnotes and code examples in their posts, too.
+
+Knowing that teams often need to write documentation for internal users, or build websites that don’t meet [the criteria for using the GOV.UK logo and GDS Transport typeface](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk), the plugin enables the following can be customised:
+
+* Organisation name and logo
+* Font family
+* Brand colour
+* Licence and copyright information
+* Site icons and Open Graph share images
+
+![Examples of customised sites using the plugin](/assets/posts/govuk-eleventy-plugin/customisation-examples.png)
+
+Sites can be published to any number of hosting platforms, including [GitHub Pages](https://pages.github.com), [Netlify](https://www.netlify.com) and [Heroku](https://www.heroku.com).
+
+## Get started
+
+The plugin is already being used by different teams across government who have built marketing pages, blogs, glossaries – and design histories, of course.
+
+Get started with the plugin by following the [instructions on the documentation site](https://x-govuk.github.io/govuk-eleventy-plugin/get-started). This project [is open source](https://github.com/x-govuk/govuk-eleventy-plugin) and contributions are welcome. If you come across an issue or have an idea for a feature, please [file an issue](https://github.com/x-govuk/govuk-eleventy-plugin/issues). We’re excited to see how you use this tool.
```

Comment:
The GitHub UI seems to use “submit”. Although “post” might be a more user friendly term?

```suggestion
Get started with the plugin by following the [instructions on the documentation site](https://x-govuk.github.io/govuk-eleventy-plugin/get-started). This project [is open source](https://github.com/x-govuk/govuk-eleventy-plugin) and contributions are welcome. If you come across an issue or have an idea for a feature, please [submit an issue](https://github.com/x-govuk/govuk-eleventy-plugin/issues). We’re excited to see how you use this tool.
```